src/neo/Cryptography/Crypto.cs

```diff
@@ -35,74 +34,26 @@ public static byte[] Sign(byte[] message, byte[] prikey, byte[] pubkey)
 
         public static bool VerifySignature(ReadOnlySpan<byte> message, ReadOnlySpan<byte> signature, ECC.ECPoint pubkey)
         {
-            if (pubkey.Curve == ECC.ECCurve.Secp256r1)
+            ECCurve curve =
+                pubkey.Curve == ECC.ECCurve.Secp256r1 ? ECCurve.NamedCurves.nistP256 :
+                pubkey.Curve == ECC.ECCurve.Secp256k1 ? ECCurve.CreateFromOid(new Oid("1.3.132.0.10")) :
```

Comment:
Is this curve enabled for all operative systems?

Reply:
I'm not sure. Can you test it on linux?

Reply:
Yes, It works on github actions, and it was in linux

```suggestion
                pubkey.Curve == ECC.ECCurve.Secp256k1 ? ECCurve.CreateFromFriendlyName("secP256k1") :
```

Reply:
Did you test this? I think there is no difference.

Reply:
Yes, I tested with `secP256k1`, It's the same but more readable